testing/pom.xml

```diff
@@ -11,7 +11,7 @@
   <properties>
     <slf4j.version>1.7.25</slf4j.version>
     <logback.version>1.2.3</logback.version>
-    <junit.version>5.6.2</junit.version>
+    <junit.version>5.7.0</junit.version>
```

Comment:
Is it just to keep up with the latest version, or do we use new junit features in this PR?

Reply:
It's for the `enabledIf` annotation that I used in `NowInSecondsTest` and `PreparedStatementAlterTableTest`.